irida_staramr_results/irida_api.py

```diff
@@ -1,41 +1,42 @@
+import json
```

Comment:
You had mentioned that you had a seperate PR coming for api changes, so I'm not going to look at this file to closely.